bluebird-promisell.js

```diff
@@ -0,0 +1,49 @@
+/* jshint esnext:true */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const Promise = require('bluebird');
+const P = require('bluebird-promisell');
+const R = require('ramda');
+const S = require('sanctuary');
+
+//    readFile :: Object -> String -> Promise Error String
+const readFile = S.flip(Promise.promisify(fs.readFile));
+
+//    readFileWithDirAndName :: String -> String -> Promise Error String
+const readFileWithDirAndName = R.curryN(2, (dir, name) => {
+  const filePath = path.join(dir, name);
+  return readFile({ encoding: 'utf8' })(filePath);
+});
+
+//    readFilesWithDirAndNames :: String -> [String] -> Promise Error [String]
+const readFilesWithDirAndNames = R.curryN(2, (dir, fileNames) => {
+  const readOneFile = readFileWithDirAndName(dir);
+  return P.traversep(readOneFile)(fileNames);
```

Comment:
Yeah, I agree. I wish I could use `R.traverse`, but Promise is not compatible with it. That's why I made the `bluebird-promisell` library. 

I guess [the Task solution](https://github.com/plaid/async-problem/blob/master/tasks.js#L34) can use R.traverse ?